tasks/Functions-client/request.js

```diff
@@ -86,32 +87,41 @@ task("functions-request", "Initiates a request from an Functions client contract
       gasLimit,
       maxPriorityFeePerGas.add(lastBaseFeePerGas)
     )
-    // Ensure the subscription has a sufficent balance
-    const linkBalance = subInfo[0]
-    if (subInfo[0].lt(estimatedCostJuels)) {
+    const estimatedCostLink = hre.ethers.utils.formatUnits(estimatedCostJuels, 18)
+
+    // Ensure that the subscription has a sufficent balance
+    const linkBalance = hre.ethers.utils.formatUnits(linkBalance, 18)
+    if (linkBalance.lt(estimatedCostLink)) {
       throw Error(
-        `Subscription ${subscriptionId} does not have sufficient funds. The estimated cost is ${estimatedCostJuels} Juels LINK, but has balance of ${linkBalance}`
+        `Subscription ${subscriptionId} does not have sufficient funds. The estimated cost is ${estimatedCostLink} LINK, but the subscription only has a balance of ${linkBalance} LINK`
       )
     }
 
-    // Print the estimated cost of the request
-    console.log(
-      `\nIf all ${gasLimit} callback gas is used, this request is estimated to cost ${hre.ethers.utils.formatUnits(
-        estimatedCostJuels,
-        18
-      )} LINK`
+    const transactionEstimateGas = await clientContract.estimateGas.executeRequest(
+      request.source,
+      request.secrets ?? [],
+      requestConfig.secretsLocation,
+      request.args ?? [],
+      subscriptionId,
+      gasLimit,
+      overrides
     )
+
+    await utils.promptTxCost(transactionEstimateGas, hre, true)
+
+    // Print the estimated cost of the request
     // Ask for confirmation before initiating the request on-chain
-    const rl = readline.createInterface({
-      input: process.stdin,
-      output: process.stdout,
+    await utils.prompt(
```

Comment:
Users will receive 2 prompts in order to initiate a request.  Can we combine these so they only have to confirm once?